src/utils.ts

```diff
@@ -6,8 +6,9 @@ import { instrument } from "./core";
 // tslint:disable
 export function guid() {
   let d = new Date().getTime();
-  if (window.performance && window.performance.now) {
```

Comment:
Minor. You might want to put this in a function as it's duplicated in several places.